bands/site_sections/bands.py

```diff
@@ -4,9 +4,11 @@
 @all_renderable()
 class Root:
     def index(self, session, id, message=''):
+        band = session.band(id)
         return {
             'message': message,
-            'band': session.band(id)
+            'band': band,
+            'sorted_checklist': sorted(c.CHECKLIST_ITEMS, key=lambda x: band.deadline_from_model(x['name'])),
```

Comment:
This is throwing an exception for Guests:
```
Traceback (most recent call last):
  File "/home/vagrant/uber/sideboard/env/lib/python3.4/site-packages/cherrypy/_cprequest.py", line 670, in respond
    response.body = self.handler()
  File "/home/vagrant/uber/sideboard/env/lib/python3.4/site-packages/cherrypy/lib/encoding.py", line 221, in __call__
    self.body = self.oldhandler(*args, **kwargs)
  File "/home/vagrant/uber/sideboard/env/lib/python3.4/site-packages/cherrypy/_cpdispatch.py", line 60, in __call__
    return self.callable(*self.args, **self.kwargs)
  File "/home/vagrant/uber/sideboard/plugins/uber/uber/decorators.py", line 252, in with_timing
    return func(*args, **kwargs)
  File "/home/vagrant/uber/sideboard/plugins/uber/uber/decorators.py", line 267, in with_session
    retval = func(*args, session=session, **kwargs)
  File "/home/vagrant/uber/sideboard/plugins/uber/uber/decorators.py", line 388, in with_restrictions
    return func(*args, **kwargs)
  File "/home/vagrant/uber/sideboard/plugins/uber/uber/decorators.py", line 328, in with_rendering
    result = func(*args, **kwargs)
  File "/home/vagrant/uber/sideboard/plugins/bands/bands/site_sections/bands.py", line 11, in index
    'sorted_checklist': sorted(c.CHECKLIST_ITEMS, key=lambda x: band.deadline_from_model(x['name'])),
TypeError: unorderable types: NoneType() < NoneType()
```

Maybe I am missing some required configuration in my deployment?

Reply:
Oh, that's interesting... I did not expect that one would need a guest deadline to avoid throwing an error! I'll add a new deadline to configspec.ini, but if you want to test before I get around to doing that, simply add `guest_info_deadline = 2018-10-10` to your `[dates]` section and it Should work.

Note to self: also add a comment in configspec.ini explaining that you need at least one date per group type!

Reply:
Update: Nope, this was just broken for any group type that didn't have all the checklist steps! Fix incoming.